src/MatrixClientPeg.js

```diff
@@ -136,13 +136,17 @@ class MatrixClientPeg {
     }
 
     _createClient(creds: MatrixClientCreds) {
+        // XXX: This is here and as a require because apparently circular dependencies
+        // are just broken in webpack (https://github.com/webpack/webpack/issues/1788)
+        const UserSettingsStore = require('./UserSettingsStore');
         const opts = {
             baseUrl: creds.homeserverUrl,
             idBaseUrl: creds.identityServerUrl,
             accessToken: creds.accessToken,
             userId: creds.userId,
             deviceId: creds.deviceId,
             timelineSupport: true,
+            forceTURN: UserSettingsStore.getLocalSetting('webRtcForceTURN', false),
```

Comment:
for the merge to develop: this becomes `SettingsStore.getValue("webRtcForceTurn")`.